### PR TITLE
Fix TCP header flags wrong data type

### PR DESCRIPTION
**Pull request by rikonaka**
As my issues https://github.com/libpnet/libpnet/issues/623 say.

### CODE REVIEW

**Comment by stappersg** [comment]
> As my issues #623 say.

Please remind `git log`

**Comment by rikonaka** [comment]
> > As my issues #623 say.
> 
> Please remind `git log`

Sorry, I don't quite understand what this is asking me to do. 🥲

**Comment by stappersg** [comment]
On Sun, Jul 09, 2023 at 07:43:39PM -0700, isinstance wrote:
> > > As my issues #623 say.
> > 
> > Please remind `git log`

Make it possible that the commit history can be read without "github.com"

 
> Sorry, I don't quite understand what this is asking me to do. 🥲
 

- Close this merge request
- Close issue #623
- Start a new merge request, make sure it has a good commit message



Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by stappersg** [comment]
> Start a new merge request, make sure it has a good commit message

That became #625
